### PR TITLE
Add Meshery in Kubernetes list

### PR DESCRIPTION
**Pull request by iArchitSharma**
As an cloud native management platform, Meshery has built-in support infastructure and application configuration management. Use Meshery to configure your multi-cluster Kubernetes environments on-premises or across clouds.

Meshery - https://meshery.io/

### CODE REVIEW

**Comment by srenatus** [comment]
How about also mentioning it at https://github.com/StyraInc/awesome-opa#built-with-wasm?

File: README.md [hunk 1]
```diff
@@ -169,6 +169,7 @@ A curated list of [awesome](https://github.com/sindresorhus/awesome) Open Policy
 
 ## Kubernetes
 
+- [Meshery](https://meshery.io/) - Use Meshery to configure your multi-cluster Kubernetes environments on-premises or across clouds.
```
Comment (anderseknert) [inline README.md:172]: Could you please change the description to say how Meshery uses OPA? That's presumably what readers of this list will want to know :) Thanks.

Comment (iArchitSharma) [reply README.md]: Done!

**Comment by iArchitSharma** [comment]
@anderseknert Sorry for the late response, i have update the description

**Review by anderseknert — APPROVED** [review]

**Pull request merged**